code/scripting/api/objs/shipclass.cpp

```diff
@@ -996,16 +996,17 @@ ADE_FUNC(isInTechroom, l_Shipclass, NULL, "Gets whether or not the ship class is
 ADE_FUNC(renderTechModel,
 	l_Shipclass,
 	"number X1, number Y1, number X2, number Y2, [number RotationPercent =0, number PitchPercent =0, number "
-	"BankPercent=40, number Zoom=1.3]",
-	"Draws ship model as if in techroom",
+	"BankPercent=40, number Zoom=1.3, number Lighting=0]",
+	"Draws ship model as if in techroom. 0 for regular lighting 1 for flat lighting.",
```

Comment:
That kinda should be a bool, and not an integer in my opinion, unless you intent to add more modes. in which case, this should be an enum

Reply:
Sure. Done.